app/moderation/ModerationPageContent.tsx

```diff
@@ -0,0 +1,1508 @@
+"use client";
+
+import React, { useState } from 'react';
+import { useSearchParams } from 'next/navigation';
+import Link from 'next/link';
```

Comment:
This should be `Link` from `reactRouterWrapper`; probably need to add that to the lint rule.  (It has saner default behavior for prefetching, among other things.)